plugins/modules/snmp_facts.py

```diff
@@ -13,7 +13,7 @@
   - Patrick Ogenstad (@ogenstad)
 short_description: Retrieve facts for a device using SNMP
 description:
-  - Retrieve facts for a device using SNMP, the facts are inserted to the C(ansible_facts) key.
+  - Retrieve facts for a device using SNMP, the facts are inserted to the RV(ignore:ansible_facts) key.
```

Comment:
The `ignore:` can be removed, once the return values are *correctly* documented, by putting them in an `ansible_facts` dict. That will also render them correctly in the HTML docs - right now they're labelled "return values" there, not "facts".